### PR TITLE
works with tests

### PR DESCRIPTION
**Pull request by aylusltd**

### CODE REVIEW

File: tests/config.yaml [hunk 1]
```diff
@@ -7,4 +7,5 @@ modules:
     dry: true
     dependencies: false
 repos:
+ - ""
  - git@github.com:StartupOS/small-test-repo.git
```
Comment (SeanTConrad) [inline tests/config.yaml:11]: Would length be 0 for this?

File: tests/test_aws.py [hunk 2]
```diff
@@ -62,3 +62,57 @@ def test_aws_scan(self):
...
+    assert agent.config.dry is False
+    assert agent.config.runGit is False
+    assert len(agent.config.modules.cloud) > 0
+    try:
+        shutil.rmtree(results_dir)
+    except Exception as e:
+        print(e)
+    assert Path(results_dir).exists() is False
+    os.makedirs(agent.config.output_dir, exist_ok=True)
+    agent.scan()
```
Comment (SeanTConrad) [inline tests/test_aws.py:95]: How will the test environment have the aws cli and access to that account?

**Review by SeanTConrad — CHANGES_REQUESTED** [review]
Just a couple of questions

**Review by SeanTConrad — APPROVED** [review]

File: tests/config.yaml (hunk 1 above)
Comment (aylusltd) [reply tests/config.yaml:11]: Yup, the quotes force yaml to interpret it as an empty string. Without the quotes it's invalid yaml.

File: tests/test_aws.py (hunk 2 above)
Comment (aylusltd) [reply tests/test_aws.py:95]: https://github.com/StartupOS/verinfast/settings/secrets/actions

Comment (aylusltd) [reply tests/test_aws.py:95]: But also we need to use the action too. https://github.com/StartupOS/verinfast/blob/main/.github/workflows/release.yml#L68-L73

**Pull request merged**